src/core/mavsdk.h

```diff
@@ -135,23 +135,22 @@ class Mavsdk {
     public:
         enum class UsageType;
 
+        /**
+         * @brief Create new Configuration via manually configured
+         * system and component ID.
+         */
         Configuration(uint8_t system_id, uint8_t component_id);
+        /**
+         * @brief Create new Configuration using a usage type.
+         * In this mode, the system and component ID will be automatically chosen.
+         */
         Configuration(UsageType usage_type);
         ~Configuration();
 
-        // Configuration(const Configuration&) = delete;
-        // Configuration& operator=(const Configuration&) = delete;
-
-        /*
-         * @brief Get the system id of this configuration
-         * @return `integer` system id, between 1 and 255
-         */
+        /** @brief Get the system id of this configuration */
```

Comment:
And the `@return`. And if I told you earlier to remove the comments, then I'm very sorry :confused:.

Reply:
No worries!